django_project/api/migrations/0003_auto_20230406_0551.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.2.18 on 2023-04-06 05:51
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0002_userapikey_allow_write'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userapikey',
+            name='max_request_per_day',
+            field=models.IntegerField(blank=True, help_text='Max request per day for the api key. If it is empty, it will use default_max_request_api preference.', null=True),
```

Comment:
Maximum allowed API requests per day for the API key. If empty, it will use the default_max_request_api preference.